exercises/E-conditionals/exercise.js

```diff
@@ -6,8 +6,18 @@
   If Daniel is a student, print out "Hi, I'm Daniel, I'm a student."
 */
 
-let name = "Daniel";
-let danielsRole = "mentor";
+function Role(name, danielsRole) {
```

Comment:
Functions usually start with a small letter.

Reply:
Yes, I overlooked it.